spring-integration-redis/src/test/java/org/springframework/integration/redis/util/RedisLockRegistryTests.java

```diff
@@ -516,19 +509,19 @@ public void concurrentObtainCapacityTest() throws InterruptedException {
 
 
 		registry.expireUnusedOlderThan(-1000);
-		assertThat(TestUtils.getPropertyValue(registry, "locks", Map.class).size()).isEqualTo(0);
+		assertThat(TestUtils.getPropertyValue(registry, "locks", Map.class).size()).isZero();
```

Comment:
`assertThat(TestUtils.getPropertyValue(registry, "locks", Map.class)).isEmpty();` ? 😃

Reply:
Yeah, I initially fixed such places with `isEmpty` but in this test class it gives compilation warning about unchecked assignments, so I decided that leaving one more `@SuppressWarnings` will not be worth it.
Should I add it anyway?

Reply:
Woah, just found that the class has already 
`org.springframework.integration.redis.util.RedisLockRegistryTests#getRedisLockRegistryLocks`
defined.
Will reuse it !

Reply:
Finally fixed!